doc/nrf/releases/release-notes-latest.rst

```diff
@@ -20,8 +20,18 @@ See the change log for each library in the :doc:`nrfxlib documentation <nrfxlib:
 sdk-zephyr
 ==========
 
+.. NOTE TO MAINTAINERS: The latest Zephyr commit appears in multiple places; make sure you update them all.
```

Comment:
@ru-fu could we use some text substitution from sphinx so we only have to type the commit once?

Reply:
Sure, you could just define it with ` .. |commit| replace:: XXX ` (either here directly or in shortcuts.txt) and then use |commit|.

Reply:
I thought about doing it that way and using a parsed-literal for the `git log` lines, but chose not to on purpose.

Doing it that way means "up to and including `4ef29b34e3`" won't have the commit in monospaced font, because substitutions don't work within double-backticks. So it will look like "up to and including 4ef29b34e3" instead, which I assume we do not want.

Please let me know if I should make this change.

Reply:
I don't see an issue with having the commit as plain text - there's no rule that says it should be monospace, and it won't impact readability.

No strong opinion from my side if we should use substitution or not, I'm okay either way.

Reply:
> Please let me know if I should make this change.

@carlescufi ping?

Reply:
I'd much rather have the commit as monospace to show this is something you can copy and paste. Let's leave this as-is.